config/application.rb

```diff
@@ -9,8 +9,7 @@
 module Tesoro
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
-    config.autoloader = :zeitwerk
+    config.load_defaults 7.0
```

Comment:
Doesn’t this mean that the "new_framework_defaults" files for 5.2, 6.0 and 6.1 can be removed?

Reply:
yes, will do